cinema/views.py

```diff
@@ -127,6 +132,38 @@
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="title",
+                type={"type": "array", "items": {"type": "number"}},
+                description="Filtering by title",
+                required=False,
+            ),
+            OpenApiParameter(
+                name="genres",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filtering by genres "
+                            "(comma-separated list of IDs)",
+                required=False,
+            ),
+            OpenApiParameter(
+                name="actors",
+                type={"type": "list", "items": {"type": "number"}},
```

Comment:
The type for the 'actors' parameter should be 'array' instead of 'list'. In OpenAPI, 'array' is the correct type for a list of items.